components/JournalForm/JournalForm.js

```diff
@@ -0,0 +1,69 @@
+import styled from "styled-components";
+
+function JournalForm({ onJournalFormInput, onJournalSubmit }) {
+  return (
+    <>
+      <StyledForm onSubmit={onJournalSubmit}>
```

Comment:
You normally want to call `event.preventDefault` in this component where the submit happens, something like this:

```suggestion
  function handleSubmit(event) {
    event.preventDefault();
    onJournalSubmit();
  }

  return (
    <>
      <StyledForm onSubmit={handleSubmit}>
```